qemu/tests/x86_cpu_model.py

```diff
@@ -72,6 +72,16 @@ def run(test, params, env):
     cmd = params["get_model_cmd"]
     out = session.cmd_output(cmd)
     if not re.search(guest_model, out):
+        check_cmd = params.get("check_cmd")
```

Comment:
I guess you need move the vulnerabilities part out of if block in L74?

Reply:
> I guess you need move the vulnerabilities part out of if block in L74?

Thanks and update.